core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java

```diff
@@ -144,14 +153,9 @@ public class RelToSqlConverter extends SqlImplementor
   public RelToSqlConverter(SqlDialect dialect) {
     super(dialect, DEFAULT_BLOAT);
     style = new QueryStyle();
-    dispatcher = ReflectUtil.createMethodDispatcher(Result.class, this, "visit",
-      RelNode.class);
-  }
-  public RelToSqlConverter(SqlDialect dialect, QueryStyle style) {
```

Comment:
Please revert back this Constructor.

Reply:
I will address suggested changes in next PR